client/contracts/truffle/README.md

```diff
@@ -0,0 +1,63 @@
+## ENS Explained
+ENS is the Ethereum Name Service, a distributed, open, and extensible naming system based on the Ethereum blockchain.
```

Comment:
Replace Ethereum references with Kowala.

Reply:
Is that necessary? We are using ENS and if someone will check the repo later and will see an ENS name, they will know what is this for

Reply:
We should not use ENS in the first place - NS should be enough.